src/components/enter-password/tests/enter-password-controller.test.ts

```diff
@@ -120,5 +123,68 @@ describe("enter password controller", () => {
       expect(res.render).to.have.been.called;
       expect(res.status).to.have.been.calledWith(HTTP_STATUS_CODES.BAD_REQUEST);
     });
+
+    it("should logout and redirect to permanently blocked when intervention is BLOCKED", async () => {
```

Comment:
reword to `should logout and redirect to permanently suspended when intervention is BLOCKED`